api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/MonitoredItemMessageApiModel.cs

```diff
@@ -45,15 +64,21 @@ public class MonitoredItemMessageApiModel {
         /// </summary>
         [JsonProperty(PropertyName = "value",
             NullValueHandling = NullValueHandling.Ignore)]
-        public JToken Value { get; set; }
-
+        public ValueApiModel Value {get; set; }
```

Comment:
I am unclear yet why the redundancy between TypeId and Value.Type.   And why the need for ValueApiModel.   Maybe do what we discussed and move completely to DataSetMessage as format, instead of this interim step?

Reply:
Ok, I'll get rid of the ValueApiModel for now and use the TypeId to transport the datatype of the value. then we will take the thorough step into the direction of DataSetMEssage format